runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargePlugin.java

```diff
@@ -311,15 +397,23 @@ private void checkDestroyWidget()
 		{
 			return;
 		}
-
 		switch (widgetDestroyItemName.getText())
 		{
 			case "Binding necklace":
 				updateBindingNecklaceCharges(MAX_BINDING_CHARGES);
 				break;
-			case "Dodgy necklace":
```

Comment:
??

Reply:
Checkdestroywidget doesnt work since it has the 'Break' option

Reply:
It should be in separate commit then. Also, I do not see any bug report for this.